index.js

```diff
@@ -22,13 +21,14 @@ function jaegerPlugin (fastify, opts, next) {
     }
   }
 
-  const tracerOptions = {
+  const defaultOptions = {
     logger: fastify.log
   }
 
-  const tracerDefaults = { state, ...opts }
-
-  const tracer = initTracer({ ...tracerConfig, ...tracerDefaults }, { ...tracerOptions, ...tracerDefaults })
+  const tracer = initTracer(
```

Comment:
Same comment as before, however I do like the new line added instead of a single line holding the entire merge.